content/release-10-13-0/devices-10-13-0-bundle/10-13-0-336.md

```diff
@@ -0,0 +1,42 @@
+---
+weight: 30
+title: Release 10.13.0.336
+layout: bundle
+---
+
+<!--10.13.0.322-10.13.0.336-->
+
+
+### Fixes
+
+<div><table ><colgroup>
+<col style="width: 15%;"><col style="width: 70%;"><col style="width: 15%;"></colgroup>
+<thead><tr>
+<th>
+Component</th>
+<th>
+Description</th>
+<th>
+Issue</th>
+</tr>
+</thead><tbody>
+
+<tr>
+<td>Device Management</td>
+<td>Fixed an issue where users could not find a location on the map based on the address they entered.</td>
+<td>MTM-49644</td>
+</tr>
+
+<tr>
+<td>Device Management</td>
+<td>In case of many active alarms of different severities, in the "Active critical alarms" widget users might have needed to press the <b>Load more</b> button several times on an empty list in order to see their active critical alarms. This is now fixed - the widget always initially shows active critical alarms if available and the<b>Load more</b> button does not load empty pages.</td>
```

Comment:
```suggestion
<td>In case of many active alarms of different severities, in the "Active critical alarms" widget users might have needed to press the <b>Load more</b> button several times on an empty list in order to see their active critical alarms. This is now fixed - the widget always initially shows active critical alarms if available and the <b>Load more</b> button does not load empty pages.</td>
```